lib/resty/aes.lua

```diff
@@ -177,6 +180,14 @@ function _M.new(self, key, salt, _cipher, _hash, hash_rounds)
         return nil
     end
 
+    if C.EVP_CIPHER_CTX_set_padding(encrypt_ctx, padding) == 0 then
```

Comment:
No need for checking the result of EVP_CIPHER_CTX_set_padding, it always returns 1:

[evp_enc.c: int EVP_CIPHER_CTX_set_padding](https://github.com/openssl/openssl/blob/5902821d81ced5e7c5db972e4b569848500940f7/crypto/evp/evp_enc.c#L546-L553)

Reply:
It maybe return 0 now:
https://github.com/openssl/openssl/blob/474294cb664c5ac5184b7fc1a3ef37214f1f2250/crypto/evp/evp_enc.c#L1032